python/pyarrow/tests/test_ipc.py

```diff
@@ -416,6 +416,15 @@ def test_pandas_serialize_round_trip_not_string_columns():
     assert_frame_equal(result, df)
 
 
+def test_serialize_pandas_no_preserve_index():
```

Comment:
Should there also be a test for `preserve_index=True`?

Reply:
done